src/kernel/mempool_persist.cpp

```diff
@@ -35,14 +35,15 @@ using fsbridge::FopenFn;
 
 namespace kernel {
 
-static const uint64_t MEMPOOL_DUMP_VERSION = 1;
+static const uint64_t MEMPOOL_DUMP_VERSION{2};
```

Comment:
Will previous versions be able to read the mempool dump version 2? `mempool.dat` suppose  to be both backward and forward-compatible between versions

Reply:
Okay, I  was asking because I see a test for that in `mempool_compatibility.py`, probably that's why CI is failing, `mempool.dat` future versions are to be backwards incompatible then we can remove the `mempool_compatibility.py` test along with this?